spot-electron/src/spot-client-log-transport/SpotClientLogTransport.js

```diff
@@ -0,0 +1,56 @@
+const { clientController, events } = require('../client-control');
+
+/**
+ * Log levels supported by the jitsi logger.
+ */
+const levels = [ 'trace', 'debug', 'info', 'log', 'warn', 'error' ];
+
+const LOG_CACHE_SIZE = 1000;
+
+/**
+ * A class
+ */
+class SpotClientLogTransport {
+    /**
+     * Creates a new instance of {@code LoggingService} and registers itself
+     * to receive logs.
+     */
+    constructor() {
+        for (const level of levels) {
+            this[level] = this.log.bind(this);
```

Comment:
Is there a need to differentiate the different levels of logging?

Reply:
yes

Reply:
I was not being clear. I meant right now only the level "log" gets logged. Do you also need to tell the client right now to send error or warn or other levels?

Reply:
I think this should support all levels, yes